packages/enzyme-mocha/src/index.js

```diff
@@ -0,0 +1,33 @@
+let jsdomify;
+
+try {
+  jsdomify = require('jsdomify');
+  jsdomify.create();
+  require('react'); // require react explicitly after jsdomify.create() has been called
+  jsdomify.destroy();
+} catch (e) {
+  // jsdom is not supported
+}
+
+export function jsdomSetup() {
+  if (!jsdomify) return;
+  jsdomify.create();
+}
+
+export function jsdomTeardown() {
+  if (!jsdomify) return;
+  jsdomify.destroy();
+}
+
+export function describeWithDOM(a, b) {
+  describe('(uses jsdom)', () => {
```

Comment:
what happens when this is imported outside a test context, and then cached with the global `describe` set to `undefined`? I think perhaps this export may need to either take `describe` as an argument, or, actually be a function that takes `describe` and returns `describeWithDOM`?